evm/src/main/java/org/hyperledger/besu/evm/code/CodeV1Validation.java

```diff
@@ -135,9 +135,9 @@ private CodeV1Validation() {
     VALID, // 0x59 - MSIZE
     VALID, // 0x5a - GAS
     VALID, // 0x5b - NOOOP (née JUMPDEST)
-    VALID_AND_TERMINAL, // 0X5c - RJUMP
-    VALID, // 0X5d - RJUMPI
-    VALID, // 0X5e - RJUMPV
+    VALID, // 0X5c - TLOAD
+    VALID, // 0X5d - TSTORE
+    INVALID, // 0X5e
```

Comment:
should this be VALID for MCOPY

Reply:
This PR was submitted before the MCOPY PR.